lang/llvm-powerpc/Portfile

```diff
@@ -0,0 +1,182 @@
+    0003-llvm-objc-Add-Apple-ObjCMetaData-library-copied-from.patch \
+    0004-llvm-Arm-Add-instruction-alias-for-vmov-R-R-l.patch \
+    0005-llvm-ARM-Make-.thumb_func-imply-.thumb.patch \
+    0006-llvm-Chi-Hua-s-strnlen-patch.patch \
+    0007-llvm-Darwin9-More-strncpy.patch \
+    0008-llvm-Darwin9-For-earlier-Darwin-round-is-in-the-glob.patch \
+    0009-llvm-Darwin9-pthread-on-D9-is-missing-pthread_setnam.patch \
+    0010-llvm-Darwin-ppc-Allow-as-PC-value.patch \
+    0011-llvm-Darwin-ppc-Use-the-same-separator-char-as-cctoo.patch \
+    0012-llvm-PowerPC-Allow-Quoted-symbol-names.patch \
+    0013-Initial-Version-on-7.0.0.patch \
+    0026-llas-Default-to-DWARF-2-for-now.patch \
+    0041-dsymutil-WIP-DEBUG-work-on-GCC-LTO-problem.patch \
+    0042-llvm-dsymutil-Mark-debug-stuff-as-debug.patch \
+    0043-llvm-ppc-Use-bcl-20-31-picbase_label-instead-of-bl.patch \
+    0044-llvm-Darwin-ppc-Improve-support-for-Mach-O-relocs.patch \
+    0045-llvm-ppc-Revise-Darwin-Directive-to-MCPU.patch \
+    0046-llvm-Darwin-Allow-CPU-SubType-override-from-assemble.patch \
+    0047-llvm-Darwin-PPC-Allow-ppc-as-an-internal-arch-name.patch \
+    0048-llvm-Add-ppc64-thread-state.patch \
+    0049-llvm-Add-ppc64-to-checkThreadCommand.patch \
+    0058-llvm-Add-support-for-force_cpusubtype_ALL-and-n.patch \
+    0059-llvm-Add-support-for-selective-disable-of-compact_un.patch \
+    0061-LLVM-Darwin-ppc-Make-DL-reflect-i1-32b-size-and-alig.patch \
+    0062-llvm-darwin-ppc-Fix-up-DL-for-powerpc-alignment-rule.patch \
+    0063-llas-Handle-no_compact_unwind.patch \
+    0064-llas-Don-t-report-support-for-compressed-debug-disab.patch \
+    0072-LLVM-ppc-Initial-impl.-of-check-for-LO14-reloc-valid.patch \
+    0073-Backports-for-AVX512-from-LLVM9.patch \
+    0074-7.1.1-PowerPC-Darwin-Update-version.patch \
+    0075-llvm-Darwin-ppc-Darwin-s-default-relocation-model-is.patch \
+
+# Extra patches
+patchfiles-append \
+    0082-Fix-for-missing-gethostuuid-on-powerpc.patch
+
+compiler.blacklist          *clang* *gcc-4.* macports-gcc-5
+
+cmake.build_type            Release
+
+configure.post_args         ../${worksrcdir}
+default configure.dir       {${workpath}/build}
+default build.dir           {${workpath}/build}
+
+cmake.install_prefix        ${sub_prefix}
+
+configure.args-delete \
+        -DCMAKE_INSTALL_NAME_DIR=${cmake.install_prefix}/lib \
+        -DCMAKE_INSTALL_RPATH=${cmake.install_prefix}/lib \
+        -DCMAKE_SYSTEM_PREFIX_PATH="${prefix}\;/usr"
+configure.args-append \
+        -DCMAKE_SYSTEM_PREFIX_PATH="${cmake.install_prefix}\;${prefix}\;/usr"
+
+configure.args-append \
+        -DLLVM_LINK_LLVM_DYLIB=ON \
+        -DLLVM_ENABLE_ASSERTIONS=OFF \
+        -DLLVM_ENABLE_BACKTRACES=OFF \
+        -DLLVM_ENABLE_RTTI=ON \
+        -DLLVM_INCLUDE_TESTS=OFF \
+        -DLLVM_INCLUDE_EXAMPLES=OFF \
+        -DLLVM_ENABLE_FFI=ON \
+        -DLLVM_BINDINGS_LIST=none \
+        -DFFI_INCLUDE_DIR=`pkg-config --cflags-only-I libffi | sed 's/-I//'` \
```

Comment:
I think you'll find this bit here doesn't actually work. the pkg-config run in the backticks never is processed. At least, that is what I concluded when I removed it from the llvm ports.

Reply:
If you run the configure, you will see (I think) that the pkg-config line never gets executed and there are no flags put in ```DFFI_INCLUDE_DIR=``` 

It's a while since I looked at it thought (three or four years ago now).